src/main/java/mate/academy/MergeSortAction.java

```diff
@@ -1,8 +1,40 @@
 package mate.academy;
 
-public class MergeSortAction {
+import java.util.Arrays;
+import java.util.concurrent.RecursiveAction;
+
+public class MergeSortAction extends RecursiveAction {
+    private final int[] array;
 
     public MergeSortAction(int[] array) {
+        this.array = array;
+    }
+
+    @Override
+    protected void compute() {
+        if (array.length > 1) {
+            int[] leftArray = Arrays.copyOfRange(array, 0, array.length / 2);
+            int[] rightArray = Arrays.copyOfRange(array, array.length / 2, array.length);
+            invokeAll(new MergeSortAction(leftArray), new MergeSortAction(rightArray));
+            merge(leftArray, rightArray);
```

Comment:
The `merge` method is called to combine `leftArray` and `rightArray`. Ensure that this method correctly modifies the original `array` to reflect the sorted order.